### DISCUSSION TITLE
$fetch.raw returns no headers

### DISCUSSION THREAD

**Discussion by christophheich** [nuxt/nuxt#16304] [Questions]
Hey, i am having an issue with $fetch.raw somehow the headers are always empty while the body, if present works fine. Someone got an idea why i am getting this issue?

![image](https://user-images.githubusercontent.com/16765979/171044587-771fd6f3-f4f8-47d2-adf9-4f93666510ec.png)
![image](https://user-images.githubusercontent.com/16765979/171044615-61783425-9175-4dbd-b19e-1e258b1de9a8.png)
![image](https://user-images.githubusercontent.com/16765979/171044646-f9b80b55-3ba0-4136-a9e0-11f6e36bf640.png)

**Comment by darklow** [comment]
Encountered same issue, anyone found workarounds?

**Reply by TheAlexLichter** [reply]
Hey! Could you kindly provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏
